scene/gui/control.cpp

```diff
@@ -3041,6 +3041,13 @@ Control *Control::make_custom_tooltip(const String &p_text) const {
 	return nullptr;
 }
 
+void Control::show_tooltip() {
+	Viewport *viewport = get_viewport();
+	if (viewport) {
+		viewport->_gui_force_show_tooltip();
+	}
+}
```

Comment:
I find it peculiar that this `Control` method just forwards the call to `Viewport`. What if instead of this, we just have a property `Viewport.show_tooltips_immediately`?

Reply:
Ah, I see why a property wouldn't work- after `_gui_force_show_tooltip` is called, the internal flag is set to `true`, which is automatically reset to `false` shortly after when the tooltip actually shows. 

I'm still not entirely convinced of the current code... I think moving the method to `Viewport` is still a better solution: `get_viewport().show_tooltip()`

Reply:
I think it helps to have a more accessible method in **Control**. There's no other methods in **Viewport** that deal with tooltips, so it would really feel like the odd one out and harm discoverability